playbook/app/pb_kits/playbook/pb_docs/copyCode.js

```diff
@@ -0,0 +1,10 @@
+const copyContent = (content) => {
+  const copyText = document.createElement("textarea")
+  copyText.value = content
+  document.body.appendChild(copyText)
+  copyText.select()
+  document.execCommand("copy")
```

Comment:
the `document.execCommand` method [has been deprecated ](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand).

What do you think of replacing this method with the `writeText` of the [clipboard api?](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API) 

```suggestion
  navigator.clipboard.writeText(content);
```